tests/SolrIndexSubsitesTest.php

```diff
@@ -221,6 +221,9 @@ public function testCorrectSubsiteIDOnPageWrite()
     public function testCorrectSubsiteIDOnFileWrite()
     {
         $subsiteIDs = ['0'] + $this->allFixtureIDs(Subsite::class);
+        $subsiteIDs = array_map(function ($v) {
```

Comment:
Can you clarify why this change is needed.

Reply:
Ensure everything is the same type.  AssertContains function got stricter.